mem-reserve

```diff
@@ -173,17 +173,33 @@ def show():
         print()
         print_mems("Reserve", reserve_map())
 
-def reserve(size):
+def reserve(size, bsize):
```

Comment:
Since you check for non-zero later, do you want to make bzie optional (ie the sig is
def reserver(size, bsize=0) ?

Reply:
Argparse for --bsize argument declares "default=0" which basically does the same thing as making the bsize parameter optional in reserve() with a default value of "0".